torchtitan/float8.py

```diff
@@ -35,9 +35,9 @@ def __init__(self, job_config: JobConfig, parallel_dims: ParallelDims):
         float8_config = job_config.float8
         if not float8_config.enable_float8_linear:
             return
-        if not _is_sm90_or_later():
+        if not _is_sm89_or_later():
             logger.warning(
-                "Failed to swap to Float8Linear because SM90 or later is not available",
+                "Failed to swap to Float8Linear because only SM89 or later is available",
```

Comment:
nit: since from your change, _what_ is available (float8 vs. compute arch) is ambiguous, maybe we can make it more direct:
```suggestion
                "Failed to swap to Float8Linear because float8 is only supported on SM89 or later",
```